### PR TITLE
fix: injected script

### PR DESCRIPTION
**Pull request by Jack-Works**
mf-3637

### CODE REVIEW

**Review by guanbinrui — DISMISSED** [review]

File: packages/scripts/src/extension/ci.ts [hunk 1]
```diff
@@ -11,9 +11,9 @@ import { type BuildFlagsExtended, getPreset, Preset } from './flags.js'
 const BUILD_PATH = new URL('build/', ROOT_PATH)
 export const ciBuild: TaskFunction = series(
     codegen,
-    function buildSandboxedPlugin() {
-        return buildSandboxedPluginConfigurable(fileURLToPath(BUILD_PATH), true)
-    },
+    // function buildSandboxedPlugin() {
+    //     return buildSandboxedPluginConfigurable(fileURLToPath(BUILD_PATH), true)
+    // },
```
Comment (guanbinrui) [inline packages/scripts/src/extension/ci.ts:16]:
```suggestion 
```

File: packages/scripts/src/bin/build.ts [hunk 2]
```diff
@@ -21,9 +21,9 @@ if (process.argv[2] === '--' || process.argv[2] === '\\--') {
     const task = series(
         //
         buildExtensionFlag('build', extensionArgsParser('production')),
-        function buildSandboxedPlugin() {
-            return buildSandboxedPluginConfigurable(fileURLToPath(new URL('./build/', ROOT_PATH)), true)
-        },
+        // function buildSandboxedPlugin() {
+        // return buildSandboxedPluginConfigurable(fileURLToPath(new URL('./build/', ROOT_PATH)), true)
+        // },
```
Comment (guanbinrui) [inline packages/scripts/src/bin/build.ts:26]:
```suggestion
```

File: packages/scripts/src/extension/ci.ts (hunk 1 above)
Comment (Jack-Works) [reply packages/scripts/src/extension/ci.ts:16]: I'd like to re-enable it in the future.

File: packages/scripts/src/bin/build.ts (hunk 2 above)
Comment (Jack-Works) [reply packages/scripts/src/bin/build.ts:26]: I'd like to re-enable it in the future.

**Pull request merged**